Makefile

```diff
@@ -20,6 +20,7 @@ android: $(BUILDDIR)/android/tun2socks.aar
 $(BUILDDIR)/android/tun2socks.aar: $(GOMOBILE)
 	mkdir -p "$(BUILDDIR)/android"
 	$(ANDROID_BUILD_CMD) -o "$@" $(IMPORT_PATH)/$(ROOT_PKG)/outline/tun2socks $(IMPORT_PATH)/$(ROOT_PKG)/outline/shadowsocks
+	unzip "$@" 'jni/*' -d $(BUILDDIR)/android
```

Comment:
Let's move the unzip to the copy files, since this is not how Makefiles are supposed to work. Unless you list all output files there. Still, in that case you would need to add another Make target that depends on the aar.

And perhaps add a TODO to investigate why we need both.